.github/workflows/ci.yml

```diff
@@ -76,11 +76,12 @@ jobs:
         pip install -r tests/requirements.txt
     - name: Test with pytest
       run: |
-        pytest --cov
+        pytest --cov --junitxml=junit.xml
     - name: Dogfooding codecov-cli
       if: ${{ !github.event.pull_request.head.repo.fork && github.repository_owner == 'codecov' }}
       run: |
         codecovcli do-upload --fail-on-error -t ${{ secrets.CODECOV_TOKEN }} --plugin pycoverage --flag python${{matrix.python-version}}
```

Comment:
This might be a stupid question, but for my own understanding, do we expect that the regular upload will also include the test result file (and won't find a parser for it and then will fail) or it's not even picked up?

Reply:
it might add it to the coverage uploads but there should not be a parser for it.

Currrent settings are that the coverage file finder will not ignore files if they have "junit" in the name [source](https://github.com/codecov/codecov-cli/blob/main/codecov_cli/services/upload/file_finder.py#L11-L177)